src/webextension/survey/components/fields/details.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
```

Comment:
uber nit: missing `;`. I should also make sure that we're running ESLint against our new .jsx friends.